common/utils/track-outbound-link.js

```diff
@@ -0,0 +1,20 @@
+import ReactGA from 'react-ga';
+
+function getDomain(url: string): string {
+  return url.replace('http://', '').replace('https://', '').split('/')[0];
```

Comment:
You could just the the URL object:
https://developer.mozilla.org/en-US/docs/Web/API/URL

and do the comparison in the `isExternal` - I know this is copy / paste, but might as well trim where we can.